handlers/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Handle freshclam service
+  ansible.builtin.service:
+    name: "{{ freshclam_service_name }}"
+    state: stopped
+  notify: Start freschlam
+
+- name: Start freschlam
+  ansible.builtin.service:
+    name: "{{ freshclam_service_name }}"
+    state: started
+
+- name: Handle clamd service
+  ansible.builtin.service:
+    name: "{{ freshclam_service_name }}"
+    state: stopped
+  notify: Start clamd
+
+- name: Start clamd
+  ansible.builtin.service:
+    name: "{{ freshclam_service_name }}"
+    state: started
```

Comment:
Why not just this?
```suggestion
- name: Restart freshclam service
  ansible.builtin.service:
    name: "{{ freshclam_service_name }}"
    state: restart

- name: Restart clamd service
  ansible.builtin.service:
    name: "{{ clamd_service_name }}"
    state: restart

Reply:
Because the restart state doesn't test that the service was started before or not. It start it whatever.